lesson06/assignment/grids.html

```diff
@@ -0,0 +1,89 @@
+        <!-- Bootstrap CSS-->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+        <title>Assignment 06 - Bootstrap 4</title>
+    </head>
+    <body>
+        <nav class= "navbar navbar-expand-sm navbar-dark bg-dark">
+            <span class="navbar-brand mb-0 h1">Navbar</span>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>            
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item"><a class="nav-link" href="index.html">Home</a></li>
+                    <li class="nav-item"><a class="nav-link" href="images.html">Images</a></li>
+                    <li class="nav-item active"><a class="nav-link" href="#">Grids<span class="sr-only">(current)</span></a></li>
+                    <li class="nav-item"><a class="nav-link" href="accordion.html">Accordion</a></li>
+                </ul>
+            </div>
+        </nav>
+        <!--basically I am just using the example in the presentation slides-->
+        <div class="row">
+          <div class="col-xl-11 col-lg-3 col-sm-6 p-5 bg-primary">
+            <button id="green-button" type="button" class="btn btn-success">Click Here!</button>
+          </div>
+          <div class="col-xl-9 col-sm-4 col-md-6 p-5 bg-warning">
+            <button id="red-button" type="button" class="btn btn-danger">Click Here!</button>
+          </div>
+        </div>
+
+        <div id="alert" class="mt-3"></div>
+
+        <script>
+        /* Had difficult time figuring out how to only have one alert out at a time, I tried using a if statement here but it doesn't seem like 
+        it works more than once though, I've left both implementations as examples on what I mean. I tried looking up some stuff
+        like using "$('.alert').show()" but I couldn't really figure out how to implement it properly. 
+        */
+            $('#green-button').click(function(e) {
```

Comment:
To make things easier, you can actually use bootstrap's "in & out" classes to make alerts pop up without actually appending any code. Here is a Stackoverflow article on this: [https://stackoverflow.com/questions/32704027/how-to-call-bootstrap-alert-with-jquery](https://stackoverflow.com/questions/32704027/how-to-call-bootstrap-alert-with-jquery)

Reply:
tried to use some google foo to find this answer, or even within the documentation for bootstrap alerts, this makes things so much easier, thank you!!